packages/data-context/src/sources/GitDataSource.ts

```diff
@@ -315,20 +318,25 @@ export class GitDataSource {
   async #getInfoWindows (absolutePaths: readonly string[]) {
     const paths = absolutePaths.map((x) => path.resolve(x)).join(',')
     const cmd = `FOR %x in (${paths}) DO (${GIT_LOG_COMMAND} %x)`
-    const result = await execa(cmd, { shell: true })
+    const result = await execa(cmd, { shell: true, cwd: this.config.projectRoot })
 
-    const split = result.stdout
-    .split('\r\n') // windows uses CRLF for carriage returns
-    .filter((str) => !str.includes('git log')) // windows stdout contains [cmd,output]. So we remove the code containing the executed command, `git log`
+    const stdout = normalize(result.stdout).split('\r\n') // windows uses CRLF for carriage returns
```

Comment:
This was the problem, `/` vs `\`. 

Def need to standardize how we handle this internally. At least w/ this PR we have more coverage and confidence across all OSes, for when we do try to standardize this (if we need to make changes, maybe we don't).

Reply:
Yeah... maybe a `guides/` would be good. Maybe a guide all about platform differences, not just paths?

In general, I think it should work like this:

1. We should always use the native path if it is being displayed to the user. Unless it's displayed in a place where it may be shared cross-platform in a programmatic way (like `require(path)`, `import path`, or in the URL bar)
2. We should always use the native path internally unless the library does not support it (like in `globby`, where `\` acts as an escape character.

What do you think?

Reply:
Guides + trusted windows tests would be a good place to start. Shared utils/libs to handle the translation when necessary could help too.